publication-commons/src/main/java/no/unit/nva/PublicationHandler.java

```diff
@@ -89,10 +89,11 @@ public static ObjectMapper createObjectMapper() {
         return new ObjectMapper()
                 .registerModule(new ProblemModule())
                 .registerModule(new JavaTimeModule())
-                .configure(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS, false)
-                .configure(SerializationFeature.INDENT_OUTPUT, true)
-                .setSerializationInclusion(JsonInclude.Include.NON_NULL)
-                .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+                .enable(DeserializationFeature.ACCEPT_EMPTY_STRING_AS_NULL_OBJECT)
+                .enable(SerializationFeature.INDENT_OUTPUT)
+                .disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES)
+                .disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS)
```

Comment:
is this for the milliseconds issue? Put is as a comment (simpler) or refactor as a fluent method

Reply:
No, this to format as date time instead of unix timestamp